src/util/SystemCommand.ts

```diff
@@ -184,31 +187,55 @@ export class SystemCommand {
                     buffer = undefined;
                 }
 
-                const condition = CommandDetection.determineErrorCondition(code, this.stdout, this.stderr);
+                const condition = CommandDetection.determineErrorCondition(code, this.stderr);
```

Comment:
**Callout**: Removing `stdout` parameter since it is not used to detect error condition.